apps/src/aichat/types.ts

```diff
@@ -28,6 +28,7 @@ export interface ChatMessage extends ChatEvent {
   chatMessageText: string;
   role: Role;
   status: ValueOf<typeof AiInteractionStatus>;
+  requestId?: number;
```

Comment:
Chat message sometimes will not have a requestId?

Reply:
When the message is first created, before it's posted to AichatCompletion it won't have one yet